plugins/modules/azure_rm_adapplication.py

```diff
@@ -433,7 +508,7 @@ def __init__(self):
             key_value=dict(type='str', no_log=True),
             native_app=dict(type='bool'),
             oauth2_allow_implicit_flow=dict(type='bool'),
-            optional_claims=dict(type='list', elements='dict', options=optional_claims_spec),
+            optional_claims=optional_claims_spec,
```

Comment:
It should be defined so that the document synchronizes changes， There is no 'complex' type!
```suggestion
            optional_claims=dict(type='list', elements='dict', options=optional_claims_spec),
```

Reply:
I'll condense this to make it clearer what I'm trying to do, I'm specifying this argument should be a dictionary, not a list